Mark9Mbugua/Politico-API
# #163735717 Intergrate Travis CI

#### What does this PR do?
It integrates Travis CI into the current project

#### Description of Task to be completed?
- Create an account on Travis CI and sync it with your GitHub account
- Create a .travis.yml file locally which will contains instructions that Travis CI will follow once it in sync 
   with your remote repository.
- Commit these changes and push to the remote repository
- Add a badge to README 

#### How should this be manually tested?
- Once commits are made to the remote repo, observe if the build on the badge is building or failing.
- Visit your profile Travis CI's website to view in details the status of the build

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2243191
#163735717
